<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selections</class>
 <widget class="QWidget" name="Selections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="tbCreate">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Create from current selection</string>
         </property>
         <property name="statusTip">
          <string>Create from current selection</string>
         </property>
         <property name="whatsThis">
          <string>Create from current selection</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/add_sel.png</normaloff>:/images/add_sel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbDelete">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Delete selection</string>
         </property>
         <property name="statusTip">
          <string>Delete selection</string>
         </property>
         <property name="whatsThis">
          <string>Delete selection</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/rem_sel.png</normaloff>:/images/rem_sel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSwitch">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Switch to selection</string>
         </property>
         <property name="statusTip">
          <string>Switch to selection</string>
         </property>
         <property name="whatsThis">
          <string>Switch to selection</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../open_iA/core/open_iA.qrc">
           <normaloff>:/images/switch_sel.png</normaloff>:/images/switch_sel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="tbAddVis">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add to Visualization</string>
           </property>
           <property name="statusTip">
            <string>Add to Visualization</string>
           </property>
           <property name="whatsThis">
            <string>Add to Visualization</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../open_iA/core/open_iA.qrc">
             <normaloff>:/images/vis_sel.png</normaloff>:/images/vis_sel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbCompare">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Compare two selections</string>
           </property>
           <property name="statusTip">
            <string>Compare two selections</string>
           </property>
           <property name="whatsThis">
            <string>Compare two selections</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../open_iA/core/open_iA.qrc">
             <normaloff>:/images/compare_sel.png</normaloff>:/images/compare_sel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../open_iA/core/open_iA.qrc"/>
 </resources>
 <connections/>
</ui>
